main.qml

```diff
@@ -99,5 +99,49 @@ Application {
             iconName: "ios-arrow-dropright"
             onClicked: if (mprisManager.canGoNext) mprisManager.next()
         }
+
+        IconButton {
+            id: volDownButton
+            edge: undefinedEdge
+            anchors.left: parent.left
+            anchors.leftMargin: Dims.w(16)
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: Dims.h(12)
+            iconName: "ios-volume-down"
+            onClicked: if (mprisManager.volume > 0.0) mprisManager.volume -= 0.05
```

Comment:
What happens if mprisManager.volume = 0.2 ? Can the value become negative? What about something like max(mprisManager.volume - 0.05, 0) (similar comment for the volUpButton)

Reply:
Good point! I've adjusted this code.
Now it still updates mprisManager.volume though even if it hasn't changed. (Not really a problem in my opinion, should be optimized in qtmpris).